site/content/docs/5.1/components/badge.md

```diff
@@ -27,7 +27,7 @@ Badges can be used as part of links or buttons to provide a counter.
 
 {{< example >}}
 <button type="button" class="btn btn-primary">
-  Notifications <span class="badge bg-secondary">4</span>
+  Notifications <span class="badge text-bg-secondary">4</span>
```

Comment:
Do we need to change the positionned example or not ?

Reply:
There is something I don't really understand in Bootstrap's modification. They use here `.text-bg-secondary` but the default value is already white. Two options:
* Remove the default color of badge with the CSS var and use everywhere in all examples `text-bg-*`
* Don't use here `.text-bg-secondary` because it doesn't bring anything
Thoughts?